src/NovelRT/Windowing/WindowingService.cpp

```diff
@@ -113,6 +115,20 @@ namespace NovelRT::Windowing {
 
 
   }
+
+  void WindowingService::checkForOptimus() {
+  #if defined (_WIN64)
+    auto optimus = LoadLibrary("nvapi64.dll");
+  #elif defined (_WIN32)
+    auto optimus = LoadLibrary("nvapi.dll");
+  #endif
+    if (optimus != nullptr) {
+      _logger.logInfoLine("NVIDIA GPU detected. Enabling...");
+    } else {
+      _logger.logWarningLine("NVIDIA GPU not detected. Continuing w/o Optimus support.");
```

Comment:
nit: This shouldn't be a warning. It isn't invalid to have an AMD GPU :P

Reply:
Good spot, lmao

Reply:
Nobody said its invalid :P I'll change it anyways 🙂

Reply:
I would generally interpret a warning as a "recoverable failure".
I wouldn't think this case is a failure, and was more just checking for optional features, in which case info is probably better.

Reply:
That makes sense